appserver/jdbc/jdbc-ra/jdbc40/pom.xml

```diff
@@ -73,9 +73,9 @@
                   Use JDBC 4.0 API regardless of which JDK we are building with.
                 -->
                 <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
+              <!--  <configuration>
```

Comment:
Instead of commenting it out, it can be removed.

Reply:
Done.